intro-HPC/title.tex

```diff
@@ -35,7 +35,7 @@
 
 Kenneth Hoste, Toon Willems, Jens Timmerman (UGent)
 
-Geert-Jan Bex, Mag Selwa (UHasselt and KULeuven)
+Geert-Jan Bex (UHasselt and KU~Leuven)
```

Comment:
Indeed, she should have been on a separate line. I've added a commit to the branch, I don't know if it is visible without resubmitting the pull request? I do see it here, so I guess it is OK.